### DISCUSSION TITLE
2D histograms with 'tile' Mark

### DISCUSSION THREAD

**Discussion by Rabeez** [mwaskom/seaborn#3083] [Q&A]
Looking through the new `Hist` implementation it seems to be specifically designed to work with a single variable. 

I'm curious what kind of API is planned for 2D histograms (didn't see them in the issues labelled with new API tag). 

Could something like this be a reasonable API to expect eventually? 
```python
(
    so.Plot(penguins, x="bill_length_mm", y="bill_depth_mm")
    .add(so.Tile(), so.Hist2D())
)
```

Since the new `Hist` opaquely uses the 'y' dimension to map counts `Hist2D` could do the same with a colorbar.

**Comment by mwaskom** [comment]
Yes it's reasonable to expect something like this eventually. I think the documentation is pretty clear in general that 

> The objects interface is currently experimental and incomplete. It is stable enough for serious use, but there certainly are some rough edges and missing features.

**Comment by Rabeez** [comment]
Would you be open to creating a discussion topic for these upcoming features so the community can bring up usecases/hacks they've had to implement in the past?
